retirement/models.py

```diff
@@ -142,6 +142,48 @@ class Meta:
         default=False
     )
 
+    google_maps_url = models.CharField(
+        blank=True,
+        null=True,
+        max_length=2000,  # Max URL length supported by IE
+        verbose_name=_("Google maps URL"),
+    )
+
+    accessibility_detail = models.TextField(
+        verbose_name=_("Accessibility Detail"),
+        null=True,
+        blank=True,
+    )
+
+    sub_title = models.CharField(
+        blank=True,
+        null=True,
+        max_length=2000,  # Max URL length supported by IE
```

Comment:
Some DB Engine could have problems with max_length > 255.
It could be better to use a TextField since only admin can populate this field.